disco_aws_automation/disco_elastigroup.py

```diff
@@ -97,6 +97,9 @@ def get_existing_groups(self, hostclass=None, group_name=None):
         try:
             groups = self._spotinst_call().json()['response']['items']
             groups = [group for group in groups if group['name'].startswith(self.environment_name)]
+        except SpotinstException as err:
+            logger.info('Unable to get existing Spotinst groups: %s', err.message)
```

Comment:
Probably better to do a `logger.exception` here to preserve the stacktrace. Also you wouldn't need the `err.message` anymore.